sdcm/cluster.py

```diff
@@ -3058,8 +3059,12 @@ def run_fullscan(self, ks_cf, loader_node, db_node):
 
         self.log.info('Fullscan for ks.cf: {}'.format(ks_cf))
 
-        result = loader_node.run_cqlsh(cmd='select count(*) from {}'.format(ks_cf),
-                                       timeout=3600, verbose=False, target_db_node=db_node)
+        try:
+            result = loader_node.run_cqlsh(cmd='select count(*) from {}'.format(ks_cf),
+                                           timeout=3600, verbose=False, target_db_node=db_node, connect_timeout=10)
+        except (UnexpectedExit, Failure) as details:
+            self.log.error('Fullscan command finished with errors: {}'.format(details))
+            result = details.result
```

Comment:
can we also send an `FullScanEvent(type='error')` to indicacted there's an error, with the details ?
i.e. we want error visible as event, and not only in the log

Reply:
I think better to make `FullScanEvent(type='finish'....` event with dynamic severity, like we have for reactor stalls DB log event
@aleksbykov can you improve the finish event and make the severity dynamic?

Reply:
Added to FullScanEvent code which change the type of event if exit_code of command is not 0.